net/radius-mac/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=radius-mac
+# Generally, our PKG_VERSION should match our PKG_SOURCE_VERSION
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/carlanton/radius-mac.git
+PKG_SOURCE_VERSION:=release-0.1
```

Comment:
Here I am missing some kind of validation of cloned repository.  There should be also PKG_MIRROR_HASH.